src/main/java/net/minecraftforge/common/PlantType.java

```diff
@@ -22,7 +22,7 @@
 import net.minecraft.util.math.BlockPos;
 import net.minecraft.world.IWorldReader;
 
-public enum PlantType
+public enum PlantType implements IExtensibleEnum
```

Comment:
Please add a todo comment like `TODO 1.16: This shouldn't be an enum, change it to something like ToolType`